.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
are you setting up `prettier` here? It's great but I don't see it in the `package.json`.

Reply:
On Vscode is unnecessary but I add on now 3e11ca0f8170e74e55e6dcb08a1e8e25cab032f0